CITATION.cff

```diff
@@ -0,0 +1,35 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: imap_processing
+message: 'If you use this software, please cite the software
+  version you used by its version-specific DOI. The DOI for
+  each version is available on Zenodo'
+type: software
+authors:
+  - name: IMAP Science Operations Center
```

Comment:
Are we supposed to put the word developers or something else to indicate it is people behind this, or is center/group the canonical way to represent that?
```suggestion
  - name: IMAP Science Operations Center Developers
```

Reply:
Updated author to IMAP Science Data Center Developers and added a blurb about the SDC in the abstract